Library/Homebrew/cmd/casks.sh

```diff
@@ -13,8 +13,11 @@ homebrew-casks() {
   if [[ -z "${HOMEBREW_NO_INSTALL_FROM_API}" &&
         -f "${HOMEBREW_CACHE}/api/cask_names.txt" ]]
   then
-    cat "${HOMEBREW_CACHE}/api/cask_names.txt"
-    echo
+    {
+      cat "${HOMEBREW_CACHE}/api/cask_names.txt"
+      echo
+      homebrew-items '*/Casks/*\.rb' '' 's|/Casks/|/|' '^homebrew/cask'
+    } | sort -u
```

Comment:
The `sort -u` is used here to sort the names again because combining the 2 lists messes up the sorting. The `-u` can probably be removed once the core taps are filtered out though because there should be no more duplicates.

Reply:
There can still be duplicates with the short names, right? It should be possible for two formulas to exist with the same name as long as they're in different taps. Not sure how big of a deal any duplication would be though.

Reply:
> It should be possible for two formulas to exist with the same name as long as they're in different taps.

Yes, this is definitely a thing that happens and is OK.